src/CoreShop/Bundle/CoreBundle/Fixtures/Data/Application/AddressIdentifierFixture.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * CoreShop.
+ *
+ * This source file is subject to the GNU General Public License version 3 (GPLv3)
+ * For the full copyright and license information, please view the LICENSE.md and gpl-3.0.txt
+ * files that are distributed with this source code.
+ *
+ * @copyright  Copyright (c) 2015-2019 Dominik Pfaffenbauer (https://www.pfaffenbauer.at)
+ * @license    https://www.coreshop.org/license     GNU General Public License version 3 (GPLv3)
+ */
+
+namespace CoreShop\Bundle\CoreBundle\Fixtures\Data\Application;
+
+use CoreShop\Bundle\FixtureBundle\Fixture\VersionedFixtureInterface;
+use CoreShop\Component\Address\Model\AddressIdentifierInterface;
+use Doctrine\Common\DataFixtures\AbstractFixture;
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+class AddressIdentifierFixture extends AbstractFixture implements ContainerAwareInterface, VersionedFixtureInterface
+{
+    /**
+     * @var ContainerInterface
+     */
+    private $container;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getVersion()
+    {
+        return '2.0';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setContainer(ContainerInterface $container = null)
+    {
+        $this->container = $container;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function load(ObjectManager $manager)
+    {
+        $fixtureData = [
+            1 => [
+                'name' => 'shipping'
+            ],
+            2 => [
+                'name' => 'invoice'
+            ]
+        ];
+
+        foreach ($fixtureData as $entry) {
+            /**
+             * @var AddressIdentifierInterface $addressIdentifier
+             */
+            $addressIdentifier = $this->container->get('coreshop.factory.address_identifier')->createNew();
+            $addressIdentifier->setName($entry['name']);
+            $addressIdentifier->setActive(true);
+            $addressIdentifier->setCreationDate(new \DateTime());
```

Comment:
the doctrine extension automatically sets the creation date, no need for that here.